akka-docs/rst/java/distributed-data.rst

```diff
@@ -63,7 +63,7 @@ will then be replicated according to the given consistency level.
 
 The ``modify`` function is called by the ``Replicator`` actor and must therefore be a pure
 function that only uses the data parameter and stable fields from enclosing scope. It must
-for example not access ``sender()`` reference of an enclosing actor.
+for example not access the ``sender`` reference of an enclosing actor.
```

Comment:
The property `sender` vs the method that you access it with, I think this is more clearly separated in Java since you cannot call methods without parenthesises, so I changed it to the name here, `getSender()` just looks weird in a sentence like this.